NOTICE.md

```diff
@@ -0,0 +1,13 @@
+Copyright (C) 2018 Sourced Technologies, S.L.
```

Comment:
I don't see this file in our guide. Could you please point why do we need it?

Reply:
Not yet merged, but agreed after https://github.com/src-d/guide/pull/305#issuecomment-436596521
It's also added as part of the `project template` by https://github.com/src-d/guide/pull/273 (not yet merged, but almost there)